GoogleForCreators/web-stories-wp
# Editor: Update Text Presets to Latest Designs

## Summary
Updates text presets to mirror changes [here](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=4180%3A292645)


## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Presets should now mirror design changes here: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=4180%3A292645

## Testing Instructions
Go to story editor & add all text presets. see that they mirror changes here: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=4180%3A292645

---



Fixes #5900
